chat/templates/chat/chat.html

```diff
@@ -0,0 +1,287 @@
+            border: none;
+            border-radius: 5px;
+            cursor: pointer;
+        }
+
+            .send-button:hover {
+                background-color: #2d0444;
+                text-decoration: none;
+                color: white
+            }
+
+        .create-button {
+            padding: 10px 20px;
+            background-color: #5B0888;
+            color: #FFFFFF;
+            border: none;
+            border-radius: 5px;
+            cursor: pointer;
+            text-decoration: none;
+        }
+
+            .create-button:hover {
+                background-color: #2d0444;
+                text-decoration: none;
+            }
+    </style>
+</head>
+
+<body>
+    {%  include 'header.html' %}
+    
+    <div class="full-container">
+        <div class="chat-sidebar">
+            <div class="chat-sidebar-header">
+                <h2>Your Chats</h2>
+                <a href="{% url 'chat:select_user_view' %}" class="create-button">New Chat</a>
+            </div>
+
+            <ul class="chat-list" id="chatList">
+                <!-- user chat list here -->
+                {% for receiver in receivers %}
+                {% if receiver.id == receiver_id %}
+                <li class="chat-list-item active" data-receiver-id="{{receiver.id}}">{{receiver.name}}</li>
+                {% else %}
+                <li class="chat-list-item" data-receiver-id="{{receiver.id}}">{{receiver.name}}</li>
+                {% endif %}
+                {% endfor %}
+            </ul>
+        </div>
+
+
+        <div class="chat" id="chatInterface">
+            <div class="chat-header">
+                <img src="https://cdn-icons-png.flaticon.com/512/3135/3135715.png" alt="Profile Picture" class="profile-picture"> 
+                <h2 class="user-name">{{receiver.name}}</h2>
+            </div>
+
+            <div id="messages">
+                <!-- chat messages here -->
+                {% for message in messages %}
+                {% if message.author == user %}
+                <div class="message-container outgoing-message">
+                    <div class="message">{{message.content}}</div>
+                </div>
+                {% else %}
+                <div class="message-container incoming-message">
+                    <div class="message">{{message.content}}</div>
+                </div>
+                {% endif %}
+                {% endfor %}
+            </div>
+            <div class="message-input-container">
+                <input type="text" id="messageInput" class="message-input" placeholder="Type a message...">
+                <button id="messageSubmit" onclick="sendMessage()" class="send-button">Send</button>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        var headerHeight = document.querySelector('header').offsetHeight;
+        var fullContainerHeight = `calc(100vh - ${headerHeight}px)`;
+        var fullContainer = document.querySelector('.full-container');
```

Comment:
querySelector will always match only the first occurrence of the class. So in this case, we could change it from a class to an id and use it as `#full-container` instead